Source/Core/Core/IOS/SDIO/SDIOSlot0.h

```diff
@@ -118,8 +118,9 @@ class SDIOSlot0 : public Device
     V2,
   };
 
-  // Number of bytes to trigger using SDHC instead of SDSC
-  static constexpr u32 SDHC_BYTES = 0x80000000;
+  // Maximum number of bytes in an SDSC card
+  // Used to trigger using SDHC instead of SDSC
+  static constexpr u32 SDSC_BYTES = 0x80000000;
```

Comment:
While you're already changing this line, I think we should prefer an u64 here.

Reply:
I'm fine with this, but just curious why this would be preferred.
I realize we're only comparing it to a u64 but I doubt compilers will treat this differently.

Reply:
Maybe also consider renaming this to `SDSC_MAX_SIZE`, which is imo clearer than just "bytes"

Reply:
Sure, that makes sense. I'm generally in favor of always using units in variable names but I guess size can be expected to be bytes.